chainercv/links/model/ssd/multibox.py

```diff
@@ -0,0 +1,90 @@
+import chainer
+import chainer.functions as F
+from chainer import initializers
+import chainer.links as L
+
+
+class Multibox(chainer.Chain):
+    """Multibox head of Single Shot Multibox Detector.
+
+    This is a head part of Single Shot Multibox Detector [#]_.
+    This link computes :obj:`loc` and :obj:`conf` from feature maps.
+    :obj:`loc` contains information of the coordinates of bounding boxes
+    and :obj:`conf` contains that of classes.
+
+    .. [#] Wei Liu, Dragomir Anguelov, Dumitru Erhan,
+       Christian Szegedy, Scott Reed, Cheng-Yang Fu, Alexander C. Berg.
+       SSD: Single Shot MultiBox Detector. ECCV 2016.
+
+    Args:
+        n_class (int): The number of classes.
```

Comment:
Should imply that the background can be included in n_class.
How about "The number of classes possibly including the background.